test/Common.cpp

```diff
@@ -67,8 +68,13 @@ std::string envOrDefaultPath(std::string const& env_name, std::string const& lib
 	if (auto path = getenv(env_name.c_str()))
 		return path;
 
-	auto const searchPath =
-	{
+	std::vector<fs::path> searchPath;
+	if (auto machType = getenv("MACHTYPE"))
+		searchPath += std::vector<fs::path>{
+			fs::path("/usr/local/lib") / machType,
```

Comment:
As discussed in the chat and suggested by @ekpyron, perhaps we should make a decision here whether we want a) system-wide installation or want to enforce a b) locally present evmone.

If we want a), then we should just use `dlopen` directly with the filename and let it search all specified paths (including `LD_(LIBRARY_)PATH`).

If we want b), then we should only allow a specific location, such as `.` and `./deps` relative to the execution path, e.g. `pwd=build`, placing `evmone.so` into `build/` and running `build/test/isoltest`.

Reply:
That being said, for the sake of merging this PR I am totally fine with the change as-is.

Reply:
> That being said, for the sake of merging this PR I am totally fine with the change as-is.

The thing is: the ``MACHTYPE`` thing doesn't work :-). So we need to do something before we can merge this :-).

Reply:
(MACHTYPE is `x86_64-pc-linux-gnu`, while the path is `x86_64-linux-gnu` -.-)